### PR TITLE
Wip mdl updater remote

### PR DESCRIPTION
**Pull request by oms4suse**
Make a new module for executing remote commands.
Add public methods cluster_quorum and cluster_status.
Add new example file with this functionality.

### CODE REVIEW

File: _modules/sesceph/__init__.py [hunk 1]
```diff
@@ -1085,3 +1095,80 @@ def ceph_version():
...
+
+
+def cluster_quorum(**kwargs):
+    """
+    Get the cluster status
+
+    CLI Example:
+
+        salt '*' sesceph.cluster_status \\
+                '[rgw.]\n\tkey = AQA/vZ9WyDwsKRAAxQ6wjGJH6WV8fDJeyzxHrg==\n\tcaps rgw = \"allow *\"\n' \\
```
Comment (theanalyst) [inline _modules/sesceph/__init__.py]: Do we always need a key as well for getting a cluster status?

Comment (theanalyst) [inline _modules/sesceph/__init__.py]: Copy past error I guess

Comment (oms4suse) [inline _modules/sesceph/__init__.py]: Yes definitely, but not a very serious one, as adding the key will not break things.

Comment (swiftgist) [inline _modules/sesceph/__init__.py]: Spelling correction
1111: Get the cluster status including health and quorum status

**Comment by swiftgist** [comment]
I read through and see the rearranging.  Looks fine.

Signed-off-by: Eric Jackson ejackson@suse.com

**Comment by oms4suse** [comment]
Added fixes as requested.

**Pull request merged**